edx_proctoring/tests/test_student_view.py

```diff
@@ -388,6 +388,24 @@ def test_get_disabled_student_view(self):
             )
         )
 
+    def test_student_response_without_credit_state(self):
+        """
+        Test that response is not None for users who are not enrolled.
+        """
+        set_runtime_service('credit', MockCreditServiceNone())
+        rendered_response = get_student_view(
+            user_id=self.user_id,
+            course_id=self.course_id,
+            content_id=self.content_id,
+            context={
+                'is_proctored': True,
+                'display_name': self.exam_name,
+                'default_time_limit_mins': 90
+            },
+            user_role='student'
+        )
+        self.assertIsNotNone(rendered_response)
```

Comment:
We should check that after the fix `get_student_view` is not raising 500.

Reply:
Sure @awaisdar001 I will change it.